# Simple message to communicate intended control. Relayed by middleman.

Header header       # Reference shenanigans

float64 forward     # forward raw control input
float64 left        # left raw control input
float64 turn        # turn raw control input
